library/src/scala/tasty/util/ShowSourceCode.scala

```diff
@@ -750,9 +750,6 @@ class ShowSourceCode[T <: Tasty with Singleton](tasty0: T) extends Show[T](tasty
             printTypeAndAnnots(tp)
             this += " "
             printAnnotation(annot)
-          case tpe @ Type.TypeRef(name, _) if name.endsWith("$") =>
-            printType(tpe)
-            this += ".type"
```

Comment:
Why the two cases below are not handled in TYPE printing as well ? Can we move the whole method `printTypeAndAnnots` to TYPE printing part, instead of being a local method inside TREE printing?